content/cybersecurity/concepts/nmap/terms/tcp-connect-scan/tcp-connect-scan.md

```diff
@@ -0,0 +1,64 @@
+---
+Title: 'TCP Connect Scan'
+Description: 'A **TCP Connect Scan** is a type of Nmap scan that establishes a complete connection to the target'
+Subjects:
+  - 'Computer Science'
+  - 'Networking'
+Tags:
+  - 'Networking'
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/scan-systems-with-nmap'
+---
+
+A **TCP Connect Scan** `-sT` is a type of Nmap scan that establishes a complete connection to the target by completing a TCP 3-way handshake. 
+Transmission Control Protocol (TCP) is a connection-oriented protocol used in modern networking, like the HTTP/HTTPS connection used to view this article.
+After Nmap's TCP connect scan completes the three way handshake, Nmap then terminates the connection.
+
+This is not to be confused with another common scan technique, *TCP Syn Scan* `-sS` which does not complete its connection to the target host. Because of this, TCP Connect Scan `-sT` will take longer and require more packets to perform.
+
+  The image below illustrates Nmap's TCP Connect Scan using the TCP 3-way handshake and then terminating the connection.
+  <!-- add an image here? https://static-assets.codecademy.com/skillpaths/learn-nmap/Performing_Basic_Network_Scans/network_scan_01.png -->
+  <!-- https://raw.githubusercontent.com/Codecademy/docs/main/media/`image-name.filetype` -->
+  ![Image](/media/network_scan_01.png)
```

Comment:
This needs to be a full path to where the image will live once this is merged.

Reply:
@Dusch4593, @yangc95 how do we feel about reusing existing Codecademy assets?

Reply:
@SSwiniarski I addressed this suggestion between lines 23 and 25. I believe the correct full path is now here and correct?
I am not opposed to omitting the image from this pull request and addressing in a different branch, provided you and the rest of the team are okay with reusing Codecademy assets?

Reply:
@SSwiniarski I think if they're reused, we should stick to the format we already have where they're stored in the media folder. I see @joesizzlin has gone ahead and followed that pattern.

I would like to see a full explanation for the linked image though. Since this is documentation, a description should be given 🙂

Reply:
@SSwiniarski, @yangc95  I am drafting up a brief description for the linked image so that way it is ready to go if we need it. 

I plan to sit on this briefly to be sure there is full agreement: 
1. the image is now linked correctly 
2. if we want to keep the image in the documentation